ptero_workflow/implementation/models/methods/shell_command.py

```diff
@@ -28,7 +30,7 @@ class ShellCommand(Method):
     }
 
     VALID_CALLBACK_TYPES = Method.VALID_CALLBACK_TYPES.union(
-            ['begun', 'error', 'execute', 'failure', 'success'])
+            ['execute', 'running', 'succeeded', 'errored', 'failed'])
```

Comment:
I thought about that, but these are actually the names of the event handlers in this class.  For example, 'execute' isn't even a status.